clang/test/dpct/cmake_migration/case_006/expected.txt

```diff
@@ -0,0 +1,7 @@
+add_executable(cudasift ${cuda_sources} ${sources} OPTIONS )
+add_executable(cudasift ${cuda_sources} ${sources} OPTIONS )
```

Comment:
According to the definition of [add_executable](https://cmake.org/cmake/help/latest/command/add_executable.html ) , OPTIONS is the invalid argument for add_executable(), it should be removed, should be removed.

Reply:
Removed.